instrumentation-core/servlet-2.2/src/main/java/io/opentelemetry/instrumentation/servlet/ServletHttpServerTracer.java

```diff
@@ -126,4 +131,12 @@ private static String getSpanName(HttpServletRequest request) {
     }
     return spanName;
   }
+
+  public void updateServerSpanNameOnce(Context attachedContext, HttpServletRequest request) {
+    if (AppServerBridge.isPresent(attachedContext)
```

Comment:
Is this `isPresent` needed here? The result of `if` is the same without it

Reply:
`isPresent` is for the cases, where there is no app-server specific tracer (and thus no AppServerBridge in the context) which actually needs this update. Without this check camel tests, for example, will fail because the name changes unexpectedly.